SpatialGDK/Source/SpatialGDK/Public/Utils/ComponentFactory.h

```diff
@@ -5,6 +5,7 @@
 #include "Interop/SpatialClassInfoManager.h"
 #include "Schema/Interest.h"
 #include "Utils/RepDataUtils.h"
+#include "Utils/SpatialLatencyTracer.h"
```

Comment:
Not sure you need this?

If TraceKey exists inside it you could declare the class in this header as I think you are just using pointers.